docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        band_members.name == band_instruments.name
</pre><p>This could be just <code>"name"</code> if you want. Not sure if you're going for explicitness here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6050/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>Nice. I don't think we were going for explicitness here.</p>